### PR TITLE
Fix: User profile copy button is not working

### LINKED ISSUE #53: [Bug]: Copy button is not working on user page
### What happened?

Copy URL button is not working, it should copy the URL path

### Please provide an bug screenshot

https://user-images.githubusercontent.com/71748675/226185468-6ae14ef1-ca21-4c7b-87d6-097b6be29a5a.mp4

### What browsers are you seeing the problem on?

_No response_

### Relevant log output

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by zt4ff**
## Pull Request Checklist

Before submitting a pull request, please make sure that your changes meet the following criteria:

- [X] Ensure that your code adheres to our coding conventions and is properly formatted.
- [X] Test your changes extensively to ensure that they do not introduce any new bugs.
- [X] Confirm that you agree to the code of conduct.

## Related Issues
Closes #53 
If this pull request is related to any issues, please list them here.

## Screenshots

If your changes include any visual elements, please provide screenshots or other visual aids to help reviewers understand your changes.

## Additional Notes

If there is anything else you would like to note about your changes or the review process, please add it here.

### CODE REVIEW

**Comment by zt4ff** [comment]
Hi maintainer! It's my first time contributing to your repo. I look forward to your response and feedback.

**Comment by Sachin-chaurasiya** [comment]
> Hi maintainer! It's my first time contributing to your repo. I look forward to your response and feedback.


Hello @zt4ff , Welcome to the @OpenSource-Journey community.

Thanks for the PR.

A few things,

- Read the contributions guidelines 
- If you want to work on any issue first ask maintainers to assign it to you, once issue is assigned to you, can start working on it and raise a PR.

**Comment by Sachin-chaurasiya** [comment]
@zt4ff, comment on this #53 issue and I will assign it to you.

File: src/pages/UserPage/UserPage.tsx [hunk 1]
```diff
@@ -162,12 +162,9 @@ const UserPage = () => {
     };
   }, [contributionData]);
 
-  useMemo(() => {
+  useEffect(() => {
     const url = `${window.location.protocol}//${window.location.host}/${userName}`;
```
Comment (ShaileshParmar11) [inline src/pages/UserPage/UserPage.tsx:166]: could you verify whether it is working or not? please also add a video of the changes in the description

**Review by ShaileshParmar11 — CHANGES_REQUESTED** [review]

**Comment by Sachin-chaurasiya** [comment]
Hello @zt4ff , Let us know if you are facing any difficulties or have questions. Thanks

**Comment by Sachin-chaurasiya** [comment]
Hello @zt4ff, Any progress on this PR?

**Comment by zt4ff** [comment]
@Sachin-chaurasiya, sorry about the late response. I was away from my devices for a while now. I am back now and work on the comment you dropped ASAP

**Comment by Sachin-chaurasiya** [comment]
> @Sachin-chaurasiya, sorry about the late response. I was away from my devices for a while now. I am back now and work on the comment you dropped ASAP

No worries @zt4ff , thanks.

**Comment by Sachin-chaurasiya** [comment]
Hello @zt4ff , Any progress?

**Comment by Sachin-chaurasiya** [comment]
@all-contributors please add @zt4ff  for Code

**Pull request merged**